Jellyfin.Plugin.AniDB/Providers/AniDB/AniDbExternalId.cs

```diff
@@ -9,7 +9,7 @@ namespace Jellyfin.Plugin.AniDB.Providers.AniDB
     public class AniDbExternalId : IExternalId
     {
         public bool Supports(IHasProviderIds item)
-            => item is Series || item is Movie;
+            => item is Series || item is Season || item is Movie;
```

Comment:
This only adds the ability to add the providerid to seasons.

You also need to add something in `AniDbSeasonProvider.cs` to actually add the providerid.
I can do it when I have time, if you want me to.